src/components/molecules/card/card.scss

```
@@ -0,0 +1,50 @@
+@use "../../variables" as *;
+.card {
+  .card-img-top {
+    position: relative;
+
+    &:hover {
+      overflow: hidden;
+      transform: scale(1.1);
+      transition: 0.2s ease;
+    }

```

```suggestion
  .card-img-top {
    position: relative;
    transition: 0.2s ease;
    &:hover {
      overflow: hidden;
      transform: scale(1.1);
    }
```
transitionをhover擬似クラスに対して指定してしまうと、カーソルを話したときにtransitionが適用されません。
上のように修正お願いします！